cdap-test/src/main/java/co/cask/cdap/test/StreamManager.java

```diff
@@ -108,10 +108,21 @@
   /**
    * Get events from the specified stream in the specified interval
    *
+   * @param startTime the start time in seconds or "now-xs" format
+   * @param endTime the end time in seconds or "now-xs" format
+   * @param limit the maximum number of events to return
+   * @return a list of stream events in the given time range
+   */
+  List<StreamEvent> getEvents(String startTime, String endTime, int limit) throws IOException;
```

Comment:
The API will be easier to use if it is:

```<T> List<StreamEvent> getEvents(T startTime, T endTime, int limit)```

With the contract as using `T.toString()` as the value for startTime and endTime. This means someone can still call it with `getEvents(0, Long.MAX_VALUE, 10);` rather then `getEvents("0", String.valueOf(Long.MAX_VALUE), 10);`